packages/compiler-ssr/__tests__/ssrComponent.spec.ts

```diff
@@ -90,6 +90,32 @@ describe('ssr: components', () => {
       `)
     })
 
+    // previously this would produce syntax error `default: _withCtx((, _push, ...)`
+    // XXX if this should error it should do better than a syntax error without emitting compiled source
+    test('empty attribute should not produce syntax error', () => {
+      expect(compile(`<foo v-slot="">foo</foo>`).code).toMatchInlineSnapshot(`
+        "const { resolveComponent: _resolveComponent, withCtx: _withCtx, createTextVNode: _createTextVNode } = require(\\"vue\\")
+        const { ssrRenderComponent: _ssrRenderComponent } = require(\\"@vue/server-renderer\\")
+
+        return function ssrRender(_ctx, _push, _parent, _attrs) {
+          const _component_foo = _resolveComponent(\\"foo\\")
+
+          _push(_ssrRenderComponent(_component_foo, _attrs, {
+            default: _withCtx((_, _push, _parent, _scopeId) => {
```

Comment:
I think the test should just match the generated code for this line, otherwise unrelated changes in codegen could break this snapshot and require manual correction. WDYT?